### ISSUE TITLE
Crash when trying to plant a fruit tree 

### ISSUE THREAD

**Issue by ManiiaKop** [anegostudios/VintageStory-Issues#2150]
**Game Version:** 1.17.1  
**Platform:** Windows   
**Modded:** Yes - only mod is Player Corpse by DArkHekRoMaNT (https://mods.vintagestory.at/list/mod?userid=8)
**SP/MP:** Multiplayer  

### Description
The game runs perfectly fine, no issue whatsoever but when trying to plant a fruit tree the game instantly crashes. I've been playing for hours without any problem, not even a stutter or lag spike. I crashed 3 times in a row, always while trying to plant the fruit tree. I don't think it's the mod since that only affects the death of players, and that has been working just fine. This crash affects every players on the server.

### How to reproduce
1. Have a fruit tree cutting
2. Use it on a dirt/grass block
3. It crashes

### Logs
```
Running on 64 bit Windows with 16 GB RAM 
Game Version: v1.17.1 (Stable)
2022-09-01 14:39:32: Critical error occurred
Loaded Mods: game@1.17.1, playercorpse@1.4.1, creative@1.17.1, survival@1.17.1
System.Collections.Generic.KeyNotFoundException: La clé donnée était absente du dictionnaire.
   à System.ThrowHelper.ThrowKeyNotFoundException()
   à System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   à Vintagestory.GameContent.BlockEntityFruitTreeBranch.InitTreeRoot(String treeType, Boolean callInitialize, ItemStack parentPlantStack) dans VSSurvivalMod\Systems\FruitTree\BEFruitTreeBranch.cs:ligne 134
   à Vintagestory.Client.NoObf.ClientWorldMap.SpawnBlockEntity(String classname, BlockPos position, ItemStack byItemStack) dans VintagestoryLib\Client\ClientWorldMap.cs:ligne 1102
   à Vintagestory.API.Common.Block.OnBlockPlaced(IWorldAccessor world, BlockPos blockPos, ItemStack byItemStack) dans VintagestoryApi\Common\Collectible\Block\Block.cs:ligne 1266
   à Vintagestory.Common.BlockAccessorBase.SetSolidBlockInternal(Int32 blockId, BlockPos pos, IWorldChunk chunk, Boolean synchronize, Boolean relight, ItemStack byItemstack) dans VintagestoryLib\Common\API\BlockAccessorBase.cs:ligne 155
   à Vintagestory.Common.BlockAccessorRelaxed.SetBlock(Int32 blockId, BlockPos pos, ItemStack byItemstack) dans VintagestoryLib\Common\API\BlockAccessorRelaxed.cs:ligne 77
   à Vintagestory.API.Common.Block.DoPlaceBlock(IWorldAccessor world, IPlayer byPlayer, BlockSelection blockSel, ItemStack byItemStack) dans VintagestoryApi\Common\Collectible\Block\Block.cs:ligne 869
   à Vintagestory.GameContent.BlockFruitTreeBranch.TryPlaceBlock(IWorldAccessor world, IPlayer byPlayer, ItemStack itemstack, BlockSelection blockSel, String& failureCode) dans VSSurvivalMod\Systems\FruitTree\BlockFruitTreeBranch.cs:ligne 123
   à Vintagestory.Client.NoObf.ClientMain.OnPlayerTryPlace(BlockSelection blockSelection, String& failureCode) dans VintagestoryLib\Client\ClientMain.cs:ligne 2053
   à Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnBlockBuild(BlockSelection blockSelection, Block onBlock, String& failureCode) dans VintagestoryLib\Client\Systems\Player\MouseInWorldInteractions.cs:ligne 704
   à Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected(Single dt) dans VintagestoryLib\Client\Systems\Player\MouseInWorldInteractions.cs:ligne 557
   à Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt) dans VintagestoryLib\Client\Systems\Player\MouseInWorldInteractions.cs:ligne 89
   à Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt) dans VintagestoryLib\Client\Util\ClientEventManager.cs:ligne 199
   à Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt) dans VintagestoryLib\Client\ClientMain.cs:ligne 1176
   à Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt) dans VintagestoryLib\Client\ClientMain.cs:ligne 1364
   à Vintagestory.Client.GuiScreenRunningGame.RenderToDefaultFramebuffer(Single dt) dans VintagestoryLib\Client\MainMenu\Screens\GuiScreenRunningGame.cs:ligne 204
   à Vintagestory.Client.ScreenManager.Render(Single dt) dans VintagestoryLib\Client\ScreenManager.cs:ligne 711
   à Vintagestory.Client.ScreenManager.OnNewFrame(Single dt) dans VintagestoryLib\Client\ScreenManager.cs:ligne 615
   à Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e) dans VintagestoryLib\Client\ClientPlatform\GameWindow.cs:ligne 132
   à System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   à OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp) dans C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:ligne 479
   à OpenTK.GameWindow.DispatchRenderFrame() dans C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:ligne 454
   à OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second) dans C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:ligne 375
   à Vintagestory.Client.ClientProgram.Start(ClientProgramArgs args, String[] rawArgs)
   à Vintagestory.ClientNative.CrashReporter.Start(ThreadStart start) dans VintagestoryLib\Client\ClientPlatform\ClientNative\CrashReporter.cs:ligne 93

Event Log entries containing Vintagestory.exe, the latest 3
==================================
{ TimeGenerated = 2022-08-31 18:11:25, Site = , Source = .NET Runtime, Message = Application : Vintagestory.exe
Version du Framework : v4.0.30319
Description : le processus a été arrêté en raison d'une exception non gérée.
Informations sur l'exception : System.OutOfMemoryException
   à OpenTK.Platform.Windows.WinInputBase.Dispose(Boolean)
   à OpenTK.Platform.Windows.WinRawInput.Dispose(Boolean)
   à OpenTK.Platform.Windows.WinInputBase.Finalize()

 }
--------------
{ TimeGenerated = 2022-08-31 17:37:48, Site = , Source = Application Hang, Message = Le programme Vintagestory.exe version 1.16.5.0 a cessé d'interagir avec Windows et a été fermé. Pour voir si plus d'informations sur le problème sont disponibles, vérifiez l'historique des problèmes dans le Panneau de configuration Sécurité et maintenance.

ID de processus : 714

Heure de début : 01d8bd81d413c0a3

Heure d'arrêt : 15

Chemin d'accès à l'application : D:\Vintagestory\Vintagestory.exe

ID de rapport : 1893e4bb-6eda-4de8-ad54-b31c567e3f78

Nom complet du package défectueux : 

ID de l'application relative à un package défectueux : 

Type de blocage : Unknown
 }
--------------
{ TimeGenerated = 2022-08-29 19:56:31, Site = , Source = Application Hang, Message = Le programme Vintagestory.exe version 1.16.5.0 a cessé d'interagir avec Windows et a été fermé. Pour voir si plus d'informations sur le problème sont disponibles, vérifiez l'historique des problèmes dans le Panneau de configuration Sécurité et maintenance.

ID de processus : 985c

Heure de début : 01d8bb9575c43986

Heure d'arrêt : 15

Chemin d'accès à l'application : D:\Vintagestory\Vintagestory.exe

ID de rapport : f0a2144d-2793-4c80-ae81-63ad52f9b7a6

Nom complet du package défectueux : 

ID de l'application relative à un package défectueux : 

Type de blocage : Unknown
 }
```

**Comment by McGravin** [comment]
I'm having the same issue, without any mods installed. Tried planting a cutting three times in a row, crashed all three times.

**Comment by Craluminum2413** [comment]
Fixed in [1.17.2](https://www.vintagestory.at/forums/topic/9406-stabler-de-jank-and-game-juice-update-1172/#comment-40205:~:text=Fixed%3A%20Game%20crashing%20when%20placing%20fruit%20tree%20cuttings)
